src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -373,6 +384,45 @@ HMICapabilitiesImpl::~HMICapabilitiesImpl() {
   delete prerecorded_speech_;
 }
 
+bool HMICapabilitiesImpl::is_hmi_capabilities_initialized() const {
```

Comment:
@dtrunov , agree. it was removed in [here](https://github.com/smartdevicelink/sdl_core/commit/ff2bdeaf8a627fc698ff83f4987921b5773a3955)